packages/workflow/src/WorkflowFormatter.ts

```diff
@@ -0,0 +1,8 @@
+import type { IWorkflowBase } from '@/index';
+
+/**
+ * Display a workflow in a readable format
+ */
+export function formatWorkflow(workflow: IWorkflowBase) {
```

Comment:
since this is only used in `cli`, we should move it there, and keep only shared code in this package.

Reply:
Good point 👍 Moved it to cli